pages/common/kubectl-run.md

```diff
@@ -0,0 +1,24 @@
+# kubectl run
+
+> Run pods in Kubernetes. Specifies pod generator to avoid deprecation error in some K8S versions.
+> More information: <https://kubernetes.io/docs/reference/generated/kubectl/kubectl-commands#run>.
+
+- Start an nginx pod and expost port 80
+
+`kubectl run --generator=run-pod/v1 nginx --image=nginx --port 80`
+
+- Run an nginx pod, setting the TEST_VAR environment variable:
```

Comment:
```suggestion
- Sawn an nginx pod and set the TEST_VAR environment variable:
```

Reply:
Actually, I think this example is fine as-is.

Reply:
Sure